Makefile

```diff
@@ -223,7 +223,7 @@ build-push-image: $(CONFIG_DOCKER_TARGET) $(CONFIG_DOCKER_TARGET_QUAY) build-ope
 	@docker push $(QUAY_REGISTRY)/$(OPERATOR_IMAGE_NAME)-$(LOCAL_ARCH):$(VERSION)
 
 multiarch-image: $(CONFIG_DOCKER_TARGET) $(CONFIG_DOCKER_TARGET_QUAY) ## Generate multiarch images for operator image.
-	@MAX_PULLING_RETRY=20 RETRY_INTERVAL=30 common/scripts/multiarch_image.sh $(ARTIFACTORYA_REGISTRY) $(OPERATOR_IMAGE_NAME) $(VERSION) $(RELEASE_VERSION)
+	# @MAX_PULLING_RETRY=20 RETRY_INTERVAL=30 common/scripts/multiarch_image.sh $(ARTIFACTORYA_REGISTRY) $(OPERATOR_IMAGE_NAME) $(VERSION) $(RELEASE_VERSION)
 	@MAX_PULLING_RETRY=20 RETRY_INTERVAL=30 common/scripts/multiarch_image.sh $(QUAY_REGISTRY) $(OPERATOR_IMAGE_NAME) $(VERSION) $(LATEST_VERSION)
```

Comment:
Should be this line, right?

Reply:
@Daniel-Fan ^^^

Reply:
@horis233 
Will `make multiarch-image` be used by CICD to build the image?
If not, we should remove `RELEASE_VERSION` tag in case it will overwrite the build by CICD like tag `3.7.2`.

And I don't think the `latest` tag matters since we have both `EUS` and continuous delivery at the same time. `latest` tag is not reliable at all.

Reply:
Oh, the command you commented out is for pushing images to the artifactory, which is required.

Reply:
Oh i see, maybe I could just comment the command for multi-arch image on `quay.io` for `latest` tag